vector/src/main/java/im/vector/app/features/ui/SharedPreferencesUiStateRepository.kt

```diff
@@ -59,10 +59,44 @@ class SharedPreferencesUiStateRepository @Inject constructor(
         }
     }
 
+    override fun storeSelectedSpace(spaceId: String?, sessionId: String) {
+        sharedPreferences.edit(true) {
+            putString("$KEY_SELECTED_SPACE@$sessionId", spaceId)
+        }
+    }
+
+    override fun storeSelectedGroup(groupId: String?, sessionId: String) {
+        sharedPreferences.edit(true) {
```

Comment:
not necessary to use `commit` here (also for the other calls)

Reply:
Done

Reply:
I change the third one